imod/mf6/utilities/clip.py

```diff
@@ -54,9 +54,13 @@ def clip_by_grid(
     package: IPointDataPackage, active: xu.UgridDataArray
 ) -> IPointDataPackage:
     """Clip PointDataPackage outside unstructured grid."""
+
+    domain_slice = get_active_domain_slice(active)
+    active_clipped = active.isel(domain_slice, missing_dims="ignore")
```

Comment:
Why is this necessary?  Wouldn't this just return active?

Reply:
The way we select data in the partitioning is with a 2D grid of inactive and active cells. The line:

```is_inside_exterior = active_clipped.grid.locate_points(points) != -1```

only checks if wells are located within the grid exterior. So inactive cells have to be removed from the grid.